code/Square Root Decomposition/MO's Algorithm/MOAlgo.cpp

```diff
@@ -0,0 +1,88 @@
+//Problem : http://www.spoj.com/problems/DQUERY/
```

Comment:
Kindly remove this link as it is not consistent with our offline mission 👍

Reply:
updated PR